### PR TITLE
Using default cleanup delay if there is a problem with the table setting

### PR DESCRIPTION
**Pull request by max-jacobs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.5] - TBD
+### Changed
+- Using default deletion delay if table parameter is configured incorrectly
```
Comment (massdosage) [inline CHANGELOG.md]: Fuuuuuullll stop!

Comment (max-jacobs) [reply CHANGELOG.md]: .!

**Comment by max-jacobs** [comment]
@massdosage 

We have a ticket to update the metrics that Beekeeper reports, so I can add this issue to that ticket? That metrics ticket will include a few improvements so would probably be better there.

Will update the readme

**Comment by massdosage** [comment]
> @massdosage
> 
> We have a ticket to update the metrics that Beekeeper reports, so I can add this issue to that ticket? That metrics ticket will include a few improvements so would probably be better there.
> 
> Will update the readme

OK, yes, please add this to the AC on that ticket and then mention it here (you might need to create a GH version of it?)

**Comment by max-jacobs** [comment]
Merging this and creating new feature here: #18

**Pull request merged**